apps/frontend/src/renderer/components/project-settings/GitHubOAuthFlow.tsx

```
@@ -386,26 +391,127 @@ export function GitHubOAuthFlow({ onSuccess, onCancel }: GitHubOAuthFlowProps) {
+                      צור טוקן אישי ב-GitHub והזן אותו כאן. זה דורש הרשאות repo.
+                    </p>
+                  </div>
+                </div>
+
+                <div className="space-y-3">
+                  <div className="space-y-2">
+                    <label className="text-sm font-medium">
+                      GitHub Personal Access Token
+                    </label>
+                    <input
+                      type="password"
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)